homeassistant/components/dsmr/sensor.py

```diff
@@ -183,12 +183,11 @@ def update_entities_telegram(telegram):
     if CONF_HOST in config:
         reader_factory = partial(
             create_tcp_dsmr_reader, config[CONF_HOST], config[CONF_PORT],
-            config[CONF_DSMR_VERSION], update_entities_telegram,
-            loop=hass.loop)
+            config[CONF_DSMR_VERSION], update_entities_telegram)
     else:
         reader_factory = partial(
             create_dsmr_reader, config[CONF_PORT], config[CONF_DSMR_VERSION],
-            update_entities_telegram, loop=hass.loop)
```

Comment:
This breaks the dsmr module. Should this be replaced with `loop=asyncio.get_event_loop()`?

Reply:
I'll revert it, my bad.

Reply:
#24127

Reply:
Reinstating this should be a temporary fix. The loop argument should always be optional (since it's going to go away anyway).

Reply:
@smurfix: it's actually caused by an upstream bug, as soon as that's fixed we can fix this properly: https://github.com/ndokter/dsmr_parser/issues/36